SimpleGrid.js

```diff
@@ -0,0 +1,152 @@
+import React, { forwardRef, memo, useMemo } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import { generateStyles } from './utils';
+import useRenderRow from './hooks/useRenderRow';
+import useDimensions from './hooks/useDimensions';
+import useRows from './hooks/useRows';
+
+
+const SimpleGrid = memo(
+  forwardRef((props, ref) => {
+    const {
+      style,
+      spacing,
+      fixed,
+      data,
+      itemDimension,
+      renderItem,
+      horizontal,
+      additionalRowStyle: externalRowStyle,
+      itemContainerStyle,
+      keyExtractor: customKeyExtractor,
+      invertedRow,
+      onItemsPerRowChange,
+      ...restProps
+    } = props;
+
+    // eslint-disable-next-line react/prop-types
+    if (props.items && !props.data) {
```

Comment:
This bit is not required. This was an Error thrown to let users know of a changed prop name in FlatList. Since SimpleGrid is only part of new major version this is not relevant